src/commands/deployments/editScmType.ts

```diff
@@ -0,0 +1,23 @@
+import * as appservice from "vscode-azureappservice";
+import { AzureTreeItem, IActionContext } from "vscode-azureextensionui";
+import { SiteTreeItem } from "../../explorer/SiteTreeItem";
+import { WebAppTreeItem } from "../../explorer/WebAppTreeItem";
+import { ext } from "../../extensionVariables";
+
+export async function editScmType(actionContext: IActionContext, node?: SiteTreeItem | appservice.DeploymentsTreeItem): Promise<void> {
+    if (!node) {
+        node = <SiteTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue);
+    } else if (node instanceof appservice.DeploymentsTreeItem) {
```

Comment:
It looks like you removed the context menu from SiteTreeItem. Do you need to check this anymore?

Reply:
Yeah because the node picker will still return a SiteTreeItem

Reply:
Then change the node picker to get the deployments tree item?